### PR TITLE
Add todo pull

### PR DESCRIPTION
**Pull request by girlpunk**
Add TODOs. All stuff I can probably manage

### CODE REVIEW

File: app/src/main/java/ie/macinnes/tvheadend/sync/EpgSyncTask.java [hunk 1]
```diff
@@ -352,6 +353,7 @@ private void handleChannelAddUpdate(@NonNull HtspMessage message) {
             ));
         }
 
+        //TODO: Why do these both do the same thing?
```
Comment (kiall) [inline app/src/main/java/ie/macinnes/tvheadend/sync/EpgSyncTask.java]: They don't do the same thing :)

The first branch (where `mInitialSyncCompleted == true`) will - once the initial sync has completed - flush every channel operation as it comes in.

The second branch (!IS_BRAVIA && mPendingChannelOps.size() >= 100) will - until the initial sync has completed - flush every 100 channel operations. Unless you're on a Sony Bravia, in that case, we wait for every channel to arrive before flushing so we can sort them correctly to workaround a Sony issue.

File: app/src/main/java/ie/macinnes/tvheadend/player/HtspDataSource.java [hunk 2]
```diff
@@ -229,6 +229,7 @@ public int read(byte[] buffer, int offset, int readLength) throws IOException {
                     Log.v(TAG, "Blocking for more data ("+mDataSourceNumber+")");
                 Thread.sleep(250);
             } catch (InterruptedException e) {
+                //TODO: Clean up and re-throw, or java will do it anway
```
Comment (kiall) [inline app/src/main/java/ie/macinnes/tvheadend/player/HtspDataSource.java]: There's nothing to clean up or rethrow here, when we're InterruptedException, it's expected we just stop waiting for data - and return 0 (as we read no data into the buffer)

File: app/src/main/java/ie/macinnes/tvheadend/player/Player.java [hunk 3]
```diff
@@ -201,6 +201,7 @@ public void pause() {
         }
     }
 
+    //TODO: Never used, check if this is still needed
```
Comment (kiall) [inline app/src/main/java/ie/macinnes/tvheadend/player/Player.java]: This is groundwork for full timeshift support :)

File: app/src/main/java/ie/macinnes/tvheadend/player/Player.java [hunk 4]
```diff
@@ -213,6 +214,7 @@ public void seek(long timeMs) {
...
```
Comment (kiall) [inline app/src/main/java/ie/macinnes/tvheadend/player/Player.java]: This is groundwork for full timeshift support :)

**Review by kiall — CHANGES_REQUESTED** [review]

**Comment by girlpunk** [comment]
Closing as nothing to do any more :)

**Pull request closed without merging**